#
# I opened a case with NTAP support on this
# $response is double-declared and will error at you under use strict
#
--- lib/perl/NetApp/NaServer-old.pm	2016-09-27 16:24:40.000000000 +0000
+++ lib/perl/NetApp/NaServer.pm	2016-10-24 05:56:54.975154221 +0000
@@ -880,7 +880,6 @@
 	my $line;
 	my $sock_err = undef;
 	my $content_length = 0;
-	my $response;
 	my $hex_length;
	
 	## Perl socket timeout has no effect during socket read.
--- lib/perl/NetApp/OCUMAPI-old.pm	2016-09-27 17:06:10.000000000 +0000
+++ lib/perl/NetApp/OCUMAPI.pm	2016-11-18 19:05:55.794856254 +0000
@@ -5375,7 +5375,7 @@
 
 sub IsArrayFlag {
     my ($self, $flag) = @_;
-    return ($flag & FIELD_ARRAY) == FIELD_ARRAY;
+    return $flag ? (($flag & FIELD_ARRAY) == FIELD_ARRAY) : 0;
 }
 
 sub IsEncryptedFlag {
